### ISSUE TITLE
ValueError while saving thumbnail: I/O operation on closed file

### ISSUE THREAD

**Issue by dmarcelino** [callowayproject/django-categories#125]
Hi,

I'm getting a "ValueError : I/O operation on closed file." when I try to save a thumbnail through Django's admin interface.

The culprit seems to be in **categories/models.py**. Changing the below line:

```width, height = get_image_dimensions(self.thumbnail.file, close=True)```

To ```close=False``` fixes the issue but I'm unsure what are the full consequences.

Is this a bug? Perhaps something that needs to be updated for django 1.10?

Full save method:

```Python
    def save(self, *args, **kwargs):
        if self.thumbnail:
            from django.core.files.images import get_image_dimensions
            import django
            if django.VERSION[1] < 2:
                width, height = get_image_dimensions(self.thumbnail.file)
            else:
                width, height = get_image_dimensions(self.thumbnail.file, close=True)
        else:
            width, height = None, None

        self.thumbnail_width = width
        self.thumbnail_height = height

        super(Category, self).save(*args, **kwargs)
```

Potential Fix:

```if django.VERSION[1] < 2 or django.VERSION[1] >= 10:```

Versions:

- django==1.10.2
- django-categories==1.4.2

**Comment by epicserve** [comment]
@dmarcelino I added pull request #128, which should fix this issue. Please check it out and let me know if it fixes the issue. Once verified, I can make a release for it.

**Comment by dmarcelino** [comment]
Just tested it and it worked fine. Thanks for such a quick turn around!

**Comment by epicserve** [comment]
@dmarcelino, just published a release 1.5.3.

**Comment by dmarcelino** [comment]
Brilliant!
